app/src/main/kotlin/io/homeassistant/companion/android/webview/WebViewPresenter.kt

```diff
@@ -80,4 +80,7 @@ interface WebViewPresenter {
     /** @return `true` if the app tried starting scanning or `false` if it was missing permissions */
     fun startScanningForImprov(): Boolean
     fun stopScanningForImprov(force: Boolean)
+
+    suspend fun discardNotificationPermission()
+    suspend fun shouldAskForNotificationPermissionIfNeeded(): Boolean
```

Comment:
```suggestion
    suspend fun shouldAskNotificationPermission(): Boolean
```

As you're 1:1 passing the result of `refsRepository.shouldAskNotificationPermission()`, let's stick to the shorter name here as well.